### PR TITLE
[ID] Update and rework The Team

### PR DESCRIPTION
**Pull request by Nivalyx**
updated translation of The Team article to reflect the recent update that was being pushed in http://github.com/ppy/osu-wiki/pull/4641

also reworked a LOT of stuffs from the original translation (the whole article basically) and significantly improved the quality/readability of it as the original text seems to be mostly translating the original English article nearly 1:1 ((as evident in "mm slider" being translated into `luncuran mm` / "bug" into `kutu` / "Mappers' Guild" into `Serikat Pembuat Map` and so on)) without considering whether the direct translation makes sense contextually or not.

### CODE REVIEW

File: wiki/People/The_Team/id.md [hunk 1]
```diff
@@ -5,93 +5,94 @@ tags:
...
+| ![][flag_MX] [Repflez](https://osu.ppy.sh/users/201392) | Pengasuh umum, pemelihara osu!wiki |
+| ![][flag_FR] [Shiro](https://osu.ppy.sh/users/113005) | Pengasuh umum |
+| ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328) | Pengembang osu!, pecinta osu!mania, penumpas *bug* |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Manajer Project Loved, penasihat bidang pengembangan komunitas, penjangkau Featured Artist |
+| ![][flag_ES] [Trosk-](https://osu.ppy.sh/users/3469385) | Pengasuh umum, penyelenggara turnamen |
+| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | Pengasuh umum, penyelenggara turnamen, pemelihara osu!wiki |
 | ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55) | Spesies yang terancam punah |
 
-## Grup pengguna
+Di samping nama-nama di atas, osu! juga memiliki [tim dukungan akun](Account_support_team) (*account support team*) yang bergerak secara terpisah untuk membantu menangani berbagai permasalahan yang bersifat pribadi.
```
Comment (hikikuman) [inline wiki/People/The_Team/id.md:40]:
```suggestion
Di samping nama-nama di atas, osu! juga memiliki [account support team](Account_support_team) (*account support team*) yang bergerak secara terpisah untuk membantu menangani berbagai permasalahan yang bersifat pribadi.
```
i think it should be `account support team` as written in the wiki/id version (https://osu.ppy.sh/wiki/id/People/The_Team/Support_Team and https://osu.ppy.sh/wiki/id/People/The_Team/Account_support_team).

Comment (Nivalyx) [reply wiki/People/The_Team/id.md:40]: ya that's why i added ``(*account support team*)`` next to the link for clarity

Comment (hikikuman) [reply wiki/People/The_Team/id.md:40]: Eh, sorry, i wasn't thorough when i checked it D:

**Review by TicClick — APPROVED** [review]
this is ready for merge, right?

**Pull request merged**